test_scripts/RC/OnRCStatus/commonOnRCStatus.lua

```diff
@@ -0,0 +1,250 @@
+---------------------------------------------------------------------------------------------------
+-- OnRCStatus common module
+---------------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.defaultProtocolVersion = 2
+config.ValidateSchema = false
+config.application1.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+config.application2.registerAppInterfaceParams.appHMIType = { "REMOTE_CONTROL" }
+
+--[[ Required Shared libraries ]]
+local commonRC = require('test_scripts/RC/commonRC')
+local test = require("user_modules/dummy_connecttest")
+
+--[[ Local Variables ]]
+local commonOnRCStatus = {}
+commonOnRCStatus.modules = {
+  "CLIMATE",
+  "RADIO",
+  "AUDIO",
+  "LIGHT",
+  "HMI_SETTINGS",
+  "SEAT"
+}
```

Comment:
@GetmanetsIrina Please only existing modules in order to not depend on other proposals

Reply:
Updated